### PR TITLE
Typo fixes and documentation fix

### PR DESCRIPTION
**Pull request by BioGeek**
This PR has:
* several typo fixes found by running [codespell](https://github.com/codespell-project/codespell) on the codebase (the exact command was `uvx codespell . --skip="*.pdf" -L smoot,copin,celcius,ue,bu,dne,ond` which ignores a few false positives and ignores pdf files).
* a fix in Example from High School Physics where there was a mismatch between the text and the code.

### CODE REVIEW

**Comment by neutrinoceros** [comment]
Thanks ! Could you add a pre-commit hook to the ensure we don't let typos creep back in in the future ?
I usually go with https://github.com/crate-ci/typos/blob/master/docs/pre-commit.md

**Comment by BioGeek** [comment]
I added a pre-commit hook (but used `codespell` because that was the one I used to find the typos) and ran `pre-commit autoupdate`.

File: .pre-commit-config.yaml [hunk 1]
```diff
@@ -5,39 +5,49 @@ ci:
...
+
+    - repo: https://github.com/codespell-project/codespell
+      rev: v2.3.0
+      hooks:
+          - id: codespell
+            name: codespell
+            description: Checks for common misspellings in text files.
+            entry: codespell --skip="*.pdf" -L smoot,copin,celcius,ue,bu,dne,ond
+            language: python
+            types: [text]
```
Comment (neutrinoceros) [inline .pre-commit-config.yaml:53]: If passing arguments is the only motivation for all this cruft I think there's a simpler way
```suggestion
            args: [--skip="*.pdf", -L, 'smoot,copin,celcius,ue,bu,dne,ond']
```

**Review by neutrinoceros — APPROVED** [review]
thanks a lot, all good now !

**Pull request merged**